collect_app/src/androidTest/java/org/odk/collect/android/AllWidgetsFormTest.java

```diff
@@ -251,7 +251,7 @@ public void testStringNumberWidget() {
     }
 
     public void testUrlWidget() {
-        Uri uri = Uri.parse("http://opendatakit.org/");
+        Uri uri = Uri.parse("http://getodk.org/");
```

Comment:
Hah! I changed the XLSForm source for this and meant to check that the XML has changed but somehow I forgot. That's why the build is failing.